pkg/command/plugin_search.go

```diff
@@ -24,25 +24,18 @@ var (
 )
 
 const searchLongDesc = `Search provides the ability to search for plugins that can be installed.
-Without an argument, the command lists all plugins currently available.
-The search command can also be used with a keyword filter to filter the
-list of available plugins. If the filter is flanked with slashes, the
-filter will be treated as a regex.
+The command lists all plugins currently available for installation.
+The search command also provides flags to limit the scope of the search.
 `
 
 func newSearchPluginCmd() *cobra.Command {
 	var searchCmd = &cobra.Command{
 		Use:               "search [keyword|/regex/]",
 		Short:             "Search for a keyword or regex in the list of available plugins",
```

Comment:
Should we also update the description in Use and Short fields as well. `keyword and regex ` ?

Reply:
Absolutely!  Thanks for noticing!